api_tests/e2e/rfc003/btc_eth-erc20/test.js

```diff
@@ -334,29 +301,29 @@ describe("RFC003: Bitcoin for ERC20", () => {
             );
         res.should.have.status(200);
         bob_redeem_action = res.body;
-
-        logger.info(
-            "Bob retrieved the following redeem parameters",
-            bob_redeem_action
-        );
     });
 
-    let bob_btc_balance_before;
-
     it("[Bob] Can execute the redeem action", async function() {
         bob_redeem_action.payload.should.include.all.keys("hex", "network");
-        bob_btc_balance_before = await bitcoin.btc_balance(bob_final_address);
-
         await bob.do(bob_redeem_action);
         await bitcoin.btc_generate();
     });
 
     it("[Bob] Should have received the alpha asset after the redeem", async function() {
-        let bob_btc_balance_after = await bitcoin.btc_balance(
+        this.timeout(10000);
+        let body = await bob.poll_comit_node_until(
+            chai,
+            bob_swap_href,
+            body => body.state.alpha_ledger.status === "Redeemed"
+        );
+        let bob_redeem_txid = body.state.alpha_ledger.redeem_tx;
+
+        let bob_satoshi_received = await bitcoin.getSatoshiTransferredTo(
+            bob_redeem_txid,
             bob_final_address
         );
-        const bob_btc_balance_expected =
-            bob_btc_balance_before + alpha_asset_amount - alpha_max_fee;
-        bob_btc_balance_after.should.be.at.least(bob_btc_balance_expected);
+        const bob_satoshi_expected = alpha_asset_amount - alpha_max_fee;
+
+        bob_satoshi_received.should.be.at.least(bob_satoshi_expected);
```

Comment:
❓ Any reason we're not exact here? We know the fee right so can't we just do an equality?

Reply:
Where can we get the fee from?

Reply:
@LLFourn Please provide further input on this.